### ISSUE TITLE
Failed: Build error in akka-http-nightly default,akka-http-nightly-node,2.12.10

### ISSUE THREAD

**Issue by patriknw** [akka/akka-http#2909]
https://jenkins.akka.io:8498/job/akka-http-nightly/773/AKKA_VERSION=default,Node=akka-http-nightly-node,SCALA_VERSION=2.12.10,jdk=AdoptOpenJDK%208/consoleFull

```
[info] Compiling 127 Scala sources and 14 Java sources to /home/jenkins/workspace/workspace/akka-http-nightly/AKKA_VERSION/default/Node/akka-http-nightly-node/SCALA_VERSION/2.12.10/jdk/AdoptOpenJDK 8/akka-http-core/target/scala-2.12/classes ...
[error] /home/jenkins/workspace/workspace/akka-http-nightly/AKKA_VERSION/default/Node/akka-http-nightly-node/SCALA_VERSION/2.12.10/jdk/AdoptOpenJDK 8/akka-http-core/src/main/scala/akka/http/impl/engine/Http2Shadow.scala:7:13: object actor is not a member of package akka
[error] import akka.actor.{ ActorSystem, ExtendedActorSystem }
[error]
```

Something completely wrong with classpath?

**Comment by raboof** [comment]
I remember seeing something about a failing disk on that machine, that sounds like it might be related?

**Comment by chbatey** [comment]
https://jenkins.akka.io:8498/job/akka-http-nightly/AKKA_VERSION=default,Node=akka-http-nightly-node,SCALA_VERSION=2.12.9,jdk=AdoptOpenJDK%208/785/

**Comment by jrudolph** [comment]
The reason is that we special cased this particular build configuration to pull up the Akka version when running the tests to find incompatibilities. This stopped running a while ago (silently) because the configuration was wrong when we updated Scala versions. Then, a while ago, I re-enabled the configuration but it seems the testing broke in the mean time.

**Comment by jrudolph** [comment]
It's actually sbt 1.3.x which broke the tests. The setup only works with coursier disabled.

**Comment by jrudolph** [comment]
Fixed in #2909.
